andrewinsoul/RoleManagementSystem
# setup environment of the application

#### What does this PR do?
This PR sets up the environment for the application

#### Description of Task to be completed?
- Install packages needed by the application
- Setup the codebase to use ES6 style with Babel transpiler
- Setup project to use Sequelize ORM

#### How should this be manually tested?
- Fetch the branch locally by running `git fetch origin setup-environment`
- cd into the server directory by running `cd ./server`
- Install all application dependencies by running `npm install`
- Start the server by running `npm start`
- Access the route `localhost:8000/`
You should see a json response resembling this:
```
{
    "status": "success",
    "message": "welcome to Role Management System API"
}
```
#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)
N/A

#### Questions:
None
